### ISSUE TITLE
PRUNING NEURAL NETWORKS AT INITIALIZATION: WHY ARE WE MISSING THE MARK?

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#132]
# TL;DR
A study that evaluated methods that prune from initialization. They are better than random, but consistently perform worse than methods that prune after training. In addition, shuffling weights within each layer or reinitializing weights results in equal or better accuracy rather than degradation. Furthermore, these can be replaced by a method that determines the rate of weight pruning per layer rather than the rate of weights to be pruned
![image](https://user-images.githubusercontent.com/24524018/95158987-e4a03600-07d7-11eb-834c-54cd6c0c55ca.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2009.08576

# Submission Dates(yyyy/mm/dd)
2020/09/18

# Authors and institutions
Jonathan Frankle, Gintare Karolina Dziugaite, Daniel M. Roy, Michael Carbin
- MIT CSAIL
- Element AI
- University of Toronto
- Vector Institute

.

# Methods

# Results

# Comments
